bip-schnorr.mediawiki

```diff
@@ -38,7 +38,7 @@ made:
 * '''Public key encoding''': Instead of using ''compressed'' 33-byte encodings of elliptic curve points which are common in Bitcoin today, public keys in this proposal are encoded as 32 bytes.
 * '''Batch verification''': The specific formulation of ECDSA signatures that is standardized cannot be verified more efficiently in batch compared to individually, unless additional witness data is added. Changing the signature scheme offers an opportunity to address this.
 
-By reusing the same curve as Bitcoin uses for ECDSA, we are able to retain existing mechanisms for choosing secret and public keys, and we avoid introducing new assumptions about elliptic curve group security.
+By reusing the same curve and hash function as Bitcoin has used for ECDSA, we are able to retain existing mechanisms for choosing private and public keys, and we avoid introducing new assumptions about the security of elliptic curves and hash functions.
```

Comment:
"has used" sounds like it's not used anymore?

(I thought that we had decided on secret vs. private key in bip-schnorr, but actually we only did that for bip-taproot/tapscript. I'm in the "secret" key camp fwiw).

Reply:
Opened PR to replace private keys with secret keys #180

Reply:
> "has used" sounds like it's not used anymore?

It's present perfect, so it should not indicate that it's not used anymore. But I can revert it, yes.

Reply:
I can still revert it, of course.